src/util/criaParametrosBusca.js

```diff
@@ -0,0 +1,34 @@
+function criaPropriedade(value) {
+  if (value) return (value = new RegExp(value.replace(',', '|'), 'i'));
+}
+function removePropUndefined(objeto) {
+  const response = { ...objeto };
+  for (let prop in response) {
+    if (response[prop] === undefined) delete response[prop];
+  }
+  return response;
+}
+function criaPropriedadeVencimento(dataInicio, dataFim) {
+  if (dataInicio || dataFim) {
```

Comment:
um dos parâmetros pode ser nulo?

Reply:
Pode sim.O parametro que for nulo não é adicionado como operador de busca.